.evergreen/evergreen.yml.in

```diff
@@ -1784,6 +1791,12 @@ buildvariants:
     tasks:
       - name: e2e_tests_linux_arm64
       - name: e2e_tests_linux_arm64_openssl3
+  - name: e2e_ubuntu2404_arm64
+    display_name: "Ubuntu 24.04 arm64 (E2E Tests)"
+    run_on: ubuntu2404-arm64-small
+    tasks:
+      - name: e2e_tests_linux_arm64
+      - name: e2e_tests_linux_arm64_openssl3
```

Comment:
I think you either need to create and use a different task name here, similar to the `e2e_tests_linux_x64_60x` and friends that is already defined in https://github.com/mongodb-js/mongosh/blob/c7645d06c01c2ef3f8c7f9c2905e31ad4994d194/.evergreen/evergreen.yml.in#L1195

or you need to wait until 8.0.0 is released, because the `e2e_tests_linux_arm64` tasks pick up the latest stable version by default

Reply:
yes i realized that as well. i think its easier to wait for 8.0. i'll move it back to the blocked state for now.